src/openloco/game_commands.h

```diff
@@ -226,11 +226,12 @@ namespace openloco::game_commands
         return do_command(31, regs) != FAILURE;
     }
 
-    inline bool do_35(thing_id_t head, uint8_t order, uint16_t orderArgument, uint32_t orderOffset)
+    inline bool do_35(thing_id_t head, uint8_t order, uint64_t orderArgument, uint32_t orderOffset)
     {
         registers regs;
         regs.bl = GameCommandFlag::apply;
-        regs.ax = order | (orderArgument << 3);
+        regs.eax = order | (orderArgument << 3);
+        regs.cx = orderArgument >> 32;
```

Comment:
`cx` is only 16 bits. Does this need to be masked, explicitly?

Reply:
I don't think so. I'm planning on refactoring this when I've finished all of the route tab as its excessive as it is copying in 48bits when vast majority of the time it should be 8bits.